packages/svelte/src/internal/client/dom/blocks/each.js

```diff
@@ -116,7 +116,7 @@ export function each(node, flags, get_collection, get_key, render_fn, fallback_f
 		var parent_node = /** @type {Element} */ (node);
 
 		anchor = hydrating
-			? set_hydrate_node(/** @type {Comment | Text} */ (parent_node.firstChild))
+			? set_hydrate_node(/** @type {Comment | Text} */ (get_first_child(parent_node)))
```

Comment:
In https://github.com/sveltejs/svelte/pull/12863#discussion_r1718713190 I was referring to these calls too. Isn't `parent_node.firstChild` preferable? Surely `get_first_child` is polymorphic (megamorphic?), given we call it with different node types

Reply:
![Screenshot 2024-08-15 at 18 36 43](https://github.com/user-attachments/assets/d5d2d4d9-f1d4-49d1-be54-ee56a24a8c82)

All the IC call-sites that use these functions come back as inlined so they're not really in the JIT code.